<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqSQRemoteSignalDialogForm</class>
 <widget class="QDialog" name="pqSQRemoteSignalDialogForm">
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Floating Point Exception Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="fpeTrapUnderflow">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Trap underflow</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="fpeTrapOverflow">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Trap overflow</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="fpeTrapDivByZero">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Trap divide by zero</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="fpeTrapInvalid">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Trap invalid</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="fpeTrapInexact">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Trap inexact</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>9</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="acccept">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Exec</string>
       </property>
       <property name="text">
        <string>accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Exec</string>
       </property>
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>acccept</sender>
   <signal>released()</signal>
   <receiver>pqSQRemoteSignalDialogForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>released()</signal>
   <receiver>pqSQRemoteSignalDialogForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
